src/Raven.Server/ServerWide/JsonDeserializationCluster.cs

```diff
@@ -113,8 +116,8 @@ public class JsonDeserializationCluster : JsonDeserializationBase
             [nameof(UpdateEtlProcessStateCommand)] = GenerateJsonDeserializationRoutine<UpdateEtlProcessStateCommand>(),
             [nameof(UpdateExternalReplicationStateCommand)] = GenerateJsonDeserializationRoutine<UpdateExternalReplicationStateCommand>(),
             [nameof(DeleteOngoingTaskCommand)] = GenerateJsonDeserializationRoutine<DeleteOngoingTaskCommand>(),
-            [nameof(PutRavenConnectionString)] = GenerateJsonDeserializationRoutine<PutRavenConnectionString>(),
-            [nameof(PutSqlConnectionString)] = GenerateJsonDeserializationRoutine<PutSqlConnectionString>(),
+            [nameof(PutRavenConnectionStringCommand)] = GenerateJsonDeserializationRoutine<PutRavenConnectionStringCommand>(),
+            [nameof(PutSqlConnectionStringCommand)] = GenerateJsonDeserializationRoutine<PutSqlConnectionStringCommand>(),
```

Comment:
Since we have (need?) the backward compatibility above (in `ClusterStateMachine.cs`), shouldn't it be here as well?

Reply:
Here we just deserialize from json value to object. Before it was the command key.

Reply:
But it will throw `KeyNotFountException` here:
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/ServerWide/ClusterStateMachine.cs#L836

once it got the legacy "PutRavenConnectionString"

Reply:
OK. So I'll remove the legacy support as this command isn't used this much.

Reply:
https://github.com/ravendb/ravendb/pull/4915